recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
@pstjohn Yes, removing some of the targets just for the sake of naming doesn't make much sense. I am not familiar with conda, but in the Debian package system one can specify a "provides" relationship between packages. This is exactly what's needed here because `ampl/mp` provides the `asl` package. If such functionality is not supported by conda, perhaps this can be simulated by making `asl` an alias to `ampl/mp`.